x-pack/plugins/security_solution/server/lib/detection_engine/routes/privileges/read_privileges_route.test.ts

```diff
@@ -8,16 +8,22 @@
 import { readPrivilegesRoute } from './read_privileges_route';
 import { serverMock, requestContextMock } from '../__mocks__';
 import { getPrivilegeRequest, getMockPrivilegesResult } from '../__mocks__/request_responses';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { elasticsearchClientMock } from 'src/core/server/elasticsearch/client/mocks';
 
 describe('read_privileges route', () => {
   let server: ReturnType<typeof serverMock.create>;
-  let { clients, context } = requestContextMock.createTools();
+  let { context } = requestContextMock.createTools();
 
   beforeEach(() => {
     server = serverMock.create();
-    ({ clients, context } = requestContextMock.createTools());
+    ({ context } = requestContextMock.createTools());
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (context.core.elasticsearch.client.asCurrentUser.transport.request as any).mockResolvedValue({
+      body: getMockPrivilegesResult(),
```

Comment:
There's no way around these `any`'s at the moment? Just a nit as in a test.

Reply:
There might be. I can try that in a follow up. I copied these from the other tests. I think jest provides mock types for us to utilize to clean this up better.

Reply:
I actually found a way so I am going to attach it to this PR